packages/runtime/src/templates/getHandler.ts

```diff
@@ -210,9 +214,9 @@ export const getHandler = ({
   // We copy the file here rather than requiring from the node module
   const { Bridge } = require("./bridge");
   const { augmentFsModule, getMaxAge, getMultiValueHeaders, getPrefetchResponse, normalizePath } = require('./handlerUtils')
-  const { getNetlifyNextServer } = require('./server')
+  const { overrideRequireHooks, applyRequireHooks } = require("./requireHooks")
```

Comment:
@orinokai, can you add some comments here explaining why we need these? I know why, but just for future all of us.

Reply:
yup, comments are here: https://github.com/netlify/next-runtime/pull/2080/commits/280ed8757f1a29e06dfa62959bee89db59c79622#diff-1049078463a83a84cfa89b835fc9959df918ece727cfd9b3af820da188048fd3R60